src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1034,15 +1036,15 @@ private void expandChannelBufferSize(Session session, TaskListener listener) {
     }
 
     /**
-     * Method copies the slave jar to the remote system.
+     * Method copies the agent jar to the remote system.
      *
      * @param listener         The listener.
-     * @param workingDirectory The directory into whihc the slave jar will be copied.
+     * @param workingDirectory The directory into whihc the agent jar will be copied.
      *
      * @throws IOException If something goes wrong.
      */
-    private void copySlaveJar(TaskListener listener, String workingDirectory) throws IOException, InterruptedException {
-        String fileName = workingDirectory + "/slave.jar";
+    private void copyAgentJar(TaskListener listener, String workingDirectory) throws IOException, InterruptedException {
+        String fileName = workingDirectory + SLASH_AGENT_JAR;
```

Comment:
Note that changing file name is a breaking change for some edge use-cases. If one passes a command prefix, he/she may be able to specify path to another JAR location. I was using this trick a lot at my previous job to specify custom Remoting versions